.eslintrc.cjs

```diff
@@ -141,6 +141,12 @@ module.exports = {
       'MethodDefinition[kind!=/[constructor|get]/]',
       'FunctionDeclaration',
       'VariableDeclarator > FunctionExpression',
+      {
+        selector:
+          'ClassDeclaration:not([id.name=/BackendIdentifierConversions|ParameterPathConversions/]) MethodDefinition[static=true]',
```

Comment:
I'd rather suppress this locally using comment (with explanation).

Reply:
Done